src/main/scala/org/broadinstitute/dsde/rawls/jobexec/WorkflowSubmissionActor.scala

```diff
@@ -147,6 +147,7 @@ trait WorkflowSubmission extends FutureSupport with LazyLogging with MethodWiths
       user.userEmail.value,
       token,
       billingProject.cromwellAuthBucketUrl,
+      s"gs://${workspace.bucketName}/${submissionId}/workflow-logs",
```

Comment:
how about calling it workflow.logs, dots are used as delimiters in wdl task names so putting a dot in a task name might not be allowed and even if it is, it is probably unwieldy.